index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <body class="bk-Color">
+      <div id="navbar">
+        <a class="active" href="#home">Home</a>
+        <a href="#projects">Projects</a>
+        <a id="profile-link" href="https://github.com/mateo951/TrialsJM" target="_blank">GitHub repo</a>
```

Comment:
- [ ] Let's make sure we should always use the appropriate tags for each element throughout our HTML structure. We could have these three links wrapped in a '<nav>' to [represent a section of a page that links to other pages](http://diveintohtml5.it/semantics.html). By doing this we make sure that our tags are much more descriptive and the semantics of our app are on point!

Reply:
Great explanation here! 💯, you even give a link reference for what you saying. Just remember kindly ask for changes as I suggested in the first comment. 😄

Reply:
Great explanation here! 💯, you even give a link reference for what you saying. Just remember kindly ask for changes as I suggested in the next comment. 😄